templates/certified/search-results.html

```diff
@@ -200,133 +207,16 @@ <h3 class="p-heading--4">No results - why not try widening your search?</h3>
-    });
-  }
-
-  function toggleVendorsList() {
-    const vendorSection = document.querySelector("#vendor-section");
-    const vendorPanel = vendorSection.querySelector(".p-accordion__panel");
-    const toggleVendorButtons = document.querySelectorAll(".p-reveal-vendors");
-    toggleVendorButtons.forEach((button) => {
-      button.addEventListener("click", () => {
-        window.scrollTo(0, 360);
-        vendorPanel.classList.toggle("is-collapsed");
-        toggleVendorButtons.forEach((button) => {
-          button.classList.toggle("u-hide");
-        });
-      });
-    });
-  }
-  toggleVendorsList();
-  
-  function toggleVersionsList() {
-    const versionSection = document.querySelector("#version-section");
-    const versionPanel = versionSection.querySelector(".p-accordion__panel");
-    const toggleVersionButtons = document.querySelectorAll(".p-reveal-versions");
-    toggleVersionButtons.forEach((button) => {
-      button.addEventListener("click", () => {
-        versionPanel.classList.toggle("is-collapsed");
-        toggleVersionButtons.forEach((button) => {
-          button.classList.toggle("u-hide");
-        });
-      });
-    });
-  }
-  toggleVersionsList();
-  
-  //hides "show all .." and "show fewer" links when the accordion is closed
-  function toggleLinks() {
-    const filterSections = document.querySelectorAll(
-      ".p-accordion__group--certified"
-    );
-    filterSections.forEach((filterSection) => {
-      const tab = filterSection.querySelector(".p-accordion__tab");
-      tab.addEventListener("click", () => {
-        const links = filterSection.querySelector(".js-toggle-links");
-        if (links) {
-          links.classList.toggle("u-hide");
-        }
-      });
-    });
-  }
-  toggleLinks();
-
-function toggleDrawer(sideNavigation, show) {
-  if (sideNavigation) {
-    if (show) {
-      sideNavigation.classList.remove('is-collapsed');
-      sideNavigation.classList.add('is-expanded');
-    } else {
-      sideNavigation.classList.remove('is-expanded');
-      sideNavigation.classList.add('is-collapsed');
-    }
-  }
-}
-
-function setupSideNavigation(sideNavigation) {
-  var toggles = [].slice.call(sideNavigation.querySelectorAll('.js-drawer-toggle'));
-
-  toggles.forEach(function (toggle) {
-    toggle.addEventListener('click', function (event) {
-      event.preventDefault();
-      var sideNav = document.getElementById(toggle.getAttribute('aria-controls'));
-
-      if (sideNav) {
-        toggleDrawer(sideNav, !sideNav.classList.contains('is-expanded'));
-      }
-    });
-  });
-}
-
-function setupSideNavigations(sideNavigationSelector) {
-  var sideNavigations = [].slice.call(document.querySelectorAll(sideNavigationSelector));
-
-  sideNavigations.forEach(setupSideNavigation);
-}
-
-setupSideNavigations('.p-side-navigation, [class*="p-side-navigation--"]');
-
 </script>
 
+<script src="{{ versioned_static('js/src/certification-results.js') }}" type="module" defer></script>
+
```

Comment:
I don't think you need `type="module"` unless you are using the Common JS `import module` syntax